core/src/createTextMaskInputElement.js

```diff
@@ -17,6 +17,56 @@ export default function createTextMaskInputElement({
   onReject,
   keepCharPositions = false
 }) {
+  // Anything that we will need to keep between `update` calls, we will store in this `state` object.
+  const previousConformedValue = emptyString
```

Comment:
You don't need this line. `previousConformedValue` key can be declared inside the `state` object.